src/Sarif.Viewer.VisualStudio.Core/SdkUiUtilities.cs

```diff
@@ -106,6 +107,33 @@ internal static T GetStoredObject<T>(string storageFileName)
             return null;
         }
 
+        /// <summary>
+        /// Reads the contents of an isolated storage file and populate to an specified object instance.
+        /// </summary>
+        /// <typeparam name="T">The type of the deserialized object.</typeparam>
+        /// <param name="storageFileName">The isolated storage file.</param>
+        /// <param name="instance">Instance to be populated.</param>
+        /// <returns>Object deserialized from isolated storage file if exists, otherwize the same instance.</returns>
+        internal static T GetStoredObject<T>(string storageFileName, T instance)
```

Comment:
>GetStoredObject [](http://example.com/codeflow?start=26&length=15)

This is almost identical to the method above. Can we combine them?

Reply:
Combined into 1 method.
The need for the `T instance` parameter is because `JsonConvert.DeserializeObject<T>(reader.ReadToEnd());` in original method  will return instance of T using default comparer.
In my case I need to deserialize a `HashSet<string>` with specified comparer `StringComparer.OrdinalIgnoreCase`, so I need to passed the instance already initialized and call `JsonConvert.PopulateObject(reader.ReadToEnd(), instance);` to deserialize.